tests/test_workflows.py

```diff
@@ -0,0 +1,71 @@
+import csv
+import json
+import logging
+
+from asnake.client import ASnakeClient
+from click.testing import CliRunner
+import pytest
+import requests_mock
+
+from asaps import models, workflows
+
+
+@pytest.fixture
+def runner():
+    return CliRunner()
+
+
+@pytest.fixture
+def as_ops():
+    client = ASnakeClient(baseurl='mock://example.com', username='test',
+                          password='test')
+    as_ops = models.AsOperations(client)
+    return as_ops
+
+
+def test_export_metadata(as_ops):
+    """Test export_metadata method."""
+    with requests_mock.Mocker() as m:
+        resource = '/repositories/0/resources/423'
+        repo_id = '0'
+        field = 'ref_id'
+        ao_uri = '/repositories/0/archival_objects/1234'
+        json_object_1 = {'record_uri': ao_uri}
+        json_object_2 = {'uri': ao_uri, 'ref_id':
+                         'a2b2c2', 'display_string': 'Sample Title'}
+        m.get(f'{resource}/tree', json=json_object_1)
+        m.get(ao_uri, json=json_object_2)
+        report_dicts = workflows.export_metadata(as_ops, resource, field,
+                                                 repo_id)
+        for report_dict in report_dicts:
+            assert report_dict['uri'] == ao_uri
+            assert report_dict['title'] == 'Sample Title'
+            assert report_dict['file_identifier'] == 'a2b2c2'
+
+
+def test_create_new_dig_objs(as_ops, caplog, runner):
+    """Test create_new_dig_objs method."""
+    caplog.set_level(logging.INFO)
+    with requests_mock.Mocker() as m:
+        with runner.isolated_filesystem():
```

Comment:
You can add the CSV as a fixture, define it in your `conftest.py`, and pass it to your tests. Again, this will allow you to use the same CSV file for multiple tests which will likely be useful in the future if it isn't already. Solenoid has examples of this with XML files.

Reply:
See above